sql/core/src/main/scala/org/apache/spark/sql/catalyst/util/ExpressionSQLBuilder.scala

```diff
@@ -67,3 +122,8 @@ class ExpressionSQLBuilder(e: Expression) {
     case _ => None
   }
 }
+
+object GeneralExpression {
+  def apply(expressions: Array[V2Expression], name: String = "UNDEFINED"): GeneralSQLExpression =
+    new GeneralSQLExpression(name, expressions)
```

Comment:
Can we deal with this in GeneralExpression's constructor?

Reply:
I'm sorry. I don't understand what you mean.

Reply:
I meant that removing this `object`, and dealing with this default argument in the constructor of `GeneralExpression` at the Java file.

Reply:
OK